packages/admin/admin/src/form/FinalFormAsyncSelect.tsx

```diff
@@ -0,0 +1,13 @@
+import { SelectProps } from "@mui/material";
+import React from "react";
+
+import { useAsyncOptionsProps } from "../hooks/useAsyncOptionsProps";
+import { FinalFormSelect, FinalFormSelectProps } from "./FinalFormSelect";
+
+export interface FinalFormAsyncSelectProps<T> extends FinalFormSelectProps<T>, Omit<SelectProps, "input"> {
```

Comment:
why omit input?

Reply:
I just copied the props from FinalFormSelect without thinking much about it... but I guess it clashes with FieldRenderProps' input.